tests/unit/s2n_cert_status_extension_test.c

```diff
@@ -170,6 +169,166 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_connection_free(conn));
     };
 
+    /* Self-talk tests */
+    if (s2n_x509_ocsp_stapling_supported() && s2n_is_tls13_fully_supported()) {
```

Comment:
That's true, although I specifically wanted to test the cert_status extension with these self-talk tests, which is only sent in TLS 1.3. If we want a self-talk test for the cert_status_response extension, I wonder if it would be better to put it somewhere else, that always negotiates TLS 1.2 even when TLS 1.3 is supported? Since this PR is only modifying OCSP stapling for TLS 1.3, I only added the cert_status self-talk tests, but if you think it would make sense to add a TLS 1.2 one as well let me know.

Reply:
Alternatively, would it make sense to move these self talk tests to a more general s2n_ocsp_test.c, and then iterate over TLS 1.2 and TLS 1.3 to always send both extensions?

Reply:
It's probably fine if you don't write a TLS1.2 one, but we should probably label them clearly. So like "TLS1.3 self-talk tests" or "Self-talk tests: TLS1.3 only" instead of just "self-talk tests".